### PR TITLE
feat: use new tokens list repo

### PR DESCRIPTION
**Pull request by alter-eggo**
# Description
This pr changes the way of downloading tokens lists. 
Now new repo will be used - https://github.com/balancer/tokenlists and tokens for all networks will be fetched from single file.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/lib/config/tokens-list/index.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    Default:
+      'https://raw.githubusercontent.com/balancer/tokenlists/main/generated/balancer.tokenlist.json',
+    Vetted:
+      'https://raw.githubusercontent.com/balancer/tokenlists/main/generated/listed-old.tokenlist.json',
+  },
+  External: [
+    'ipns://tokens.uniswap.org',
+    'https://www.gemini.com/uniswap/manifest.json',
+  ],
+};
```
Comment (garethfuller) [inline src/lib/config/tokens-list/index.ts:14]: I get why you've done this but please don't change the way the config works in this PR, just replace the existing URLs with these ones in all the different network files for tokenlists.

**Review by garethfuller — CHANGES_REQUESTED** [review]

**Review by garethfuller — CHANGES_REQUESTED** [review]

File: src/services/token-list/token-list.service.ts [hunk 2]
```diff
@@ -75,15 +89,17 @@ export default class TokenListService {
...
+      }
+      if (protocol === 'ipns') {
         return await this.ipfs.get<TokenList>(path, protocol);
-      } else {
-        console.error('Unhandled TokenList protocol', uri);
-        throw new Error('Unhandled TokenList protocol');
       }
+
+      console.error('Unhandled TokenList protocol', uri);
+      throw new Error('Unhandled TokenList protocol');
```
[the remark this answers is not in the document]
Comment (alter-eggo) [reply src/services/token-list/token-list.service.ts:102]: just small refactoring to improve readability

File: src/lib/config/arbitrum/tokenlists.ts [hunk 3]
```diff
@@ -3,9 +3,9 @@ import { TokenListURLMap } from '@/types/TokenList';
 const tokenlists: TokenListURLMap = {
   Balancer: {
     Default:
-      'https://raw.githubusercontent.com/balancer/assets/refactor-for-multichain/generated/arbitrum.listed.tokenlist.json',
+      'https://raw.githubusercontent.com/balancer/tokenlists/main/generated/balancer.tokenlist.json',
     Vetted:
-      'https://raw.githubusercontent.com/balancer/assets/refactor-for-multichain/generated/arbitrum.vetted.tokenlist.json',
+      'https://raw.githubusercontent.com/balancer/tokenlists/main/generated/listed-old.tokenlist.json',
```
[the remark this answers is not in the document]
Comment (alter-eggo) [reply src/lib/config/arbitrum/tokenlists.ts]: fixed

File: src/lib/config/polygon/tokenlists.ts [hunk 4]
```diff
@@ -3,9 +3,9 @@ import { TokenListURLMap } from '@/types/TokenList';
 const tokenlists: TokenListURLMap = {
   Balancer: {
     Default:
-      'https://raw.githubusercontent.com/balancer/tokenlists/main/generated/balancer.tokenlist.json',
+      'https://raw.githubusercontent.com/balancer/tokenlists/main/generated/listed-old.tokenlist.json ',
```
Comment (garethfuller) [inline src/lib/config/polygon/tokenlists.ts:6]: Each of the URLs have this space at the end of the string, please remove.

**Review by garethfuller — CHANGES_REQUESTED** [review]

File: src/lib/config/polygon/tokenlists.ts (hunk 4 above)
Comment (alter-eggo) [reply src/lib/config/polygon/tokenlists.ts]: ah( fixed

**Review by garethfuller — CHANGES_REQUESTED** [review]
Could you figure out why Synthetix doesn't have an icon here? I checked on production and it does. RPL doesn't have an icon on production so you can ignore that.
Screenshot 2023-03-28 at 14 05 13

**Pull request merged**